pxtpy/pydecompiler.ts

```diff
@@ -45,7 +45,7 @@ namespace pxt.py {
     ///
     export const INDENT = "    "
     export function indent(lvl: number): (s: string) => string {
-        return s => `${INDENT.repeat(lvl)}${s}`
+        return s => (s || "").split('\n').map(line => `${INDENT.repeat(lvl)}${line}`).join('\n')
```

Comment:
we use indent everywhere, so this could break things if we were dependent on the previous behavior for multiline strings--i don't think we were, and i tested jumpy platformer, cat jumper, raptor run, and castle crawler (just converting to python and playing the game) and they all seemed to work, but would love a double-check from any of the python people

Reply:
I don't *think* it matters

Reply:
Seems like a good solution to me!